quest/util/misc.py

```diff
@@ -18,30 +17,43 @@
     import json
 
 
-def generate_cache(update=False):
-    """Downloads features for all providers and caches results.
+def _abs_path(path, mkdir=True):
+    """Gets the absolute path for a file to be within the Quest directory,
+    and will create a directory of that filename.
 
     Args:
-        update (bool):
-            whether to update cached files.
+        param1 (string): A string that is a filename.
```

Comment:
The doc strings should use the actual argument name (i.e. `path` and `mkdir`) not `param1` and `param2`